lottie/src/main/java/com/airbnb/lottie/L.java

```diff
@@ -74,15 +74,17 @@ private static LottieTrace getTrace() {
   }
 
   public static void setFetcher(LottieNetworkFetcher customFetcher) {
+    boolean changing = !fetcher.equals(customFetcher);
```

Comment:
I think one extra step of robustness would be to allow null here so that if you set it, you can also unset it.

Reply:
I also don't think you need to set changing. You can do
```
if (fetcher == null == customFetcher == null || (fetcher == null ? false : fetcher.equals(customFetcher)) {
  return;
}
```
Personally, I like guards that return early so you don't have to nest the rest of the business logic in an if.

Reply:
Done!